.github/workflows/precommit.yml

```diff
@@ -76,9 +76,35 @@ jobs:
           name: wheel
           path: dist/
 
+  changes:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      snowpark_pandas: ${{ steps.filter.outputs.snowpark_pandas }}
+      snowpark: ${{ steps.filter.outputs.snowpark }}
+    steps:
+      - uses: dorny/paths-filter@v3
```

Comment:
We may not be able to use this action for this repository, I previously tried couple of those actions, but seems not registered with snowflake repository. The worst case, we might need to have a script and uses github command, we can see how this action works

Reply:
Seems it was blocked 😢